### PR TITLE
Add computed `volume_path` attribute to `databricks_volume` resource

### PR DESCRIPTION
**Pull request by alexott**
## Changes

This attribute will expose path in form of `/Volumes/<catalog>/<schema>/<name>` for use with other resources, such as, `databricks_file`.  Plus it's necessary for Terraform Exporter to detect references between resources.

## Tests

- [x] `make test` run locally
- [x] relevant change in `docs/` folder
- [ ] covered with integration tests in `internal/acceptance`
- [ ] relevant acceptance tests are passing
- [ ] using Go SDK

### CODE REVIEW

File: catalog/resource_volume.go [hunk 1]
```diff
@@ -82,7 +87,11 @@ func ResourceVolume() common.Resource {
 			if err != nil {
 				return err
 			}
-			return common.StructToData(v, s, d)
+			err = common.StructToData(v, s, d)
+			if err != nil {
+				return err
+			}
+			return d.Set("volume_path", "/Volumes/"+strings.ReplaceAll(v.FullName, ".", "/"))
```
Comment (nkvuong) [inline catalog/resource_volume.go:94]: we also need to update this in Update, as volume name could change

Comment (alexott) [reply catalog/resource_volume.go:94]: done, thank you!

**Review by nkvuong — APPROVED** [review]

File: catalog/resource_volume.go [hunk 2]
```diff
@@ -42,6 +42,11 @@ func ResourceVolume() common.Resource {
 	s := common.StructToSchema(VolumeInfo{},
 		func(m map[string]*schema.Schema) map[string]*schema.Schema {
 			m["storage_location"].DiffSuppressFunc = ucDirectoryPathSlashAndEmptySuppressDiff
+			m["volume_path"] = &schema.Schema{
+				Type:     schema.TypeString,
+				Computed: true,
+				Optional: true,
```
Comment (mgyucht) [inline catalog/resource_volume.go:48]: Only question: if this is optional, we shouldn't set this if a user specifies it. I think it probably shouldn't be optional, right?

**Review by mgyucht — COMMENTED** [review]
One question, but otherwise LGTM.

File: catalog/resource_volume.go (hunk 2 above)
Comment (alexott) [reply catalog/resource_volume.go]: yeah, that's correct - it was more kind of copy/paste - we have that code in multiple places. Corrected to be computed only

**Pull request merged**